### PR TITLE
[CDAP-15659]Add link to navigate to pipeline from control center pipeline card

### PR DESCRIPTION
**Pull request by yukiej**
JIRA: https://issues.cask.co/browse/CDAP-15659
BUILD: https://builds.cask.co/browse/CDAP-UDUT388

### CODE REVIEW

File: cdap-ui/app/cdap/components/EntityCard/EntityCard.scss [hunk 1]
```diff
@@ -189,6 +189,16 @@ $jump-button-container-size: 80px;
       }
     }
   }
+
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/EntityCard/EntityCard.scss]: nit: @yukiej I think this newline might give a stylelint error. Can you rebase against latest develop and run a build?

**Review by ajainarayanan — APPROVED** [review]
Just a minor comment. Otherwise 👍 LGTM

File: cdap-ui/app/cdap/components/EntityCard/EntityCard.scss (hunk 1 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/EntityCard/EntityCard.scss]: Just fixed and rebased - thanks!

File: cdap-ui/app/cdap/components/EntityCard/index.js [hunk 2]
```diff
@@ -110,6 +115,56 @@ export default class EntityCard extends Component {
...
+            onSuccess={this.props.onFastActionSuccess}
+            className="btn-group"
+          />
+        </div>
+      </Card>
+    );
+
+    if (this.props.entity.isHydrator) {
+      const namespace = getCurrentNamespace();
+      const link = `/pipelines/ns/${namespace}/view/${this.props.entity.id}`;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/EntityCard/index.js]: I would suggest getting this link using url-generator that we have to go to `hydrator.detail`

File: cdap-ui/app/cdap/components/EntityCard/index.js [hunk 3]
```diff
@@ -110,6 +115,56 @@ export default class EntityCard extends Component {
         successMessage={this.state.successMessage}
       />
     );
+
+    let card = (
+      <Card
+        header={header}
+        id={classnames(
```
Comment (itsanudeep) [inline cdap-ui/app/cdap/components/EntityCard/index.js]: You don't need classnames here, right?

File: cdap-ui/app/cdap/components/EntityCard/index.js [hunk 4]
```diff
@@ -110,6 +115,56 @@ export default class EntityCard extends Component {
...
+              'with-version': this.props.entity.version,
+            })}
+          >
+            <h4
+              className={classnames({ 'with-version': this.props.entity.version })}
+              title={this.props.entity.id}
+            >
+              {this.props.entity.id}
+            </h4>
+            <small>{this.props.entity.version ? this.props.entity.version : ''}</small>
```
Comment (itsanudeep) [inline cdap-ui/app/cdap/components/EntityCard/index.js]: you can use objectQuery here

**Comment by yukiej** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-UDUT388-4
RAT check build: https://builds.cask.co/browse/CDAP-DRC9443-5

File: cdap-ui/app/cdap/components/EntityCard/index.js [hunk 5]
```diff
@@ -110,6 +116,63 @@ export default class EntityCard extends Component {
...
+              'with-version': this.props.entity.version,
+            })}
+          >
+            <h4
+              className={classnames({ 'with-version': this.props.entity.version })}
+              title={this.props.entity.id}
+            >
+              {this.props.entity.id}
+            </h4>
+            {/* <small>{this.props.entity.version ? this.props.entity.version : ''}</small> */}
```
Comment (itsanudeep) [inline cdap-ui/app/cdap/components/EntityCard/index.js]: I think you missed removing this

Comment (yukiej) [reply cdap-ui/app/cdap/components/EntityCard/index.js]: Thanks!

**Pull request merged**